scripts/signup.py

```diff
@@ -17,15 +18,30 @@ def signup(args):
     else:
         print(response.text)
 
+def register_producer(args):
+    headers = get_headers(None)
+    url = URL + f'_db/{DB_NAME}/{MOUNT}/producer/register'
+    body = {}
+    response = requests.post(url, json=body, headers=headers)
+    if response.status_code != 200:
+        print(f"Error: {response.status_code} {response.text}")
+    else:
+        print(response.text)
+
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
-    parser.add_argument('-u', '--user', action='store', required=True,
+    subparsers = parser.add_subparsers(help="sub-command help")
+    parser_user = subparsers.add_parser("user", help="user signup")
+    parser_user.set_defaults(func=signup)
+    parser_producer = subparsers.add_parser("producer", help="producer signup")
```

Comment:
It's supposed that producer also locks some funds.

Reply:
added this logic in the app